src/jit/CMakeLists.txt

```diff
@@ -79,6 +79,120 @@ set( JIT_SOURCES
   valuenum.cpp
 )
 
+if (WIN32)
+  set( JIT_HEADERS
+    _typeinfo.h
+    alloc.h
+    arraystack.h
+    bitset.h
+    bitsetasshortlong.h
+    bitsetasuint64.h
+    bitsetasuint64inclass.h
+    bitsetops.h
+    bitvec.h
+    block.h
+    blockset.h
+    codegen.h
+    codegenclassic.h
+    codegeninterface.h
+    codegenlinear.h
+    compiler.h
+    compiler.hpp
+    compilerbitsettraits.h
+    compilerbitsettraits.hpp
+    compmemkind.h
+    compphases.h
+    dataflow.h
+    decomposelongs.h
+    disasm.h
+    emit.h
+    emitarm.h
+    emitarm64.h
+    emitdef.h
+    emitfmts.h
+    emitfmtsarm.h
+    emitfmtsarm64.h
+    emitfmtsxarch.h
+    emitinl.h
+    emitjmps.h
+    emitpub.h
+    emitxarch.h
+    error.h
+    fp.h
+    gentree.h
+    gtlist.h
+    gtstructs.h
+    hashbv.h
+    host.h
+    hostallocator.h
+    hwintrinsiclistxarch.h
+    ICorJitInfo_API_names.h
+    ICorJitInfo_API_wrapper.hpp
+    inline.h
+    inlinepolicy.h
+    instr.h
+    instrs.h
+    instrsarm.h
+    instrsarm64.h
```

Comment:
Some headers are platform specific:
```
#if defined(_TARGET_XARCH_)
#include "instrsxarch.h"
#elif defined(_TARGET_ARM_)
#include "instrsarm.h"
#elif defined(_TARGET_ARM64_)
#include "instrsarm64.h"
#else
```

I think it is better to have all headers included rather than don't have them at all.
However, we should check that we do not want to add platform conditions here.

Reply:
> However, we should check that we do not want to add platform conditions here.

As far as I have been able to check having headers from other platforms does not interfere with intellisense and obviously has no impact on build